Dockerfile

```diff
@@ -101,6 +102,12 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install --no-instal
     texlive-latex-recommended \
     texlive-science
 
+# Install required packages for testing to be able to run
+RUN pip install -r requirements.txt -r requirements-dev.txt
+
+# Jenkins runs the containers with the `-u 1000:1000` option, not as root.
+RUN chown -Rf +1000:+1000 /venv
```

Comment:
`-f` is documented as "suppresses most error messages". Don't we want to know if something has gone wrong?

I'd never seen the leading +'s before, but I see the documentation does allow it and it ensures it's interpreted numerically even if there is a user called '1000'.

Reply:
Fair point about the `-f`.

I read through the `man` page for `chown` and couldn't see how to do it in a good way, so I will admit I asked an LLM, and it gave me the + notation. I did check it before committing though.